tests/YamlTest.php

```diff
@@ -37,7 +37,13 @@ public function testYaml()
 
         $pattern = $config['install_dir'] . '/includes/definitions/*.yaml';
         foreach (glob($pattern) as $file) {
-            $data = Yaml::parse(file_get_contents($file));
+            try {
+                $data = Yaml::parse(file_get_contents($file));
+            } catch (ParseException $e) {
+                //$this->assertNotEmpty($data, "$file Could not be parsed");
+                $this->assertEmpty($e->getMessage(), "$file Could not be parsed");
```

Comment:
Perhaps fail another way here.
`throw new PHPUnitException("$file Could not be parsed");`
maybe...